### DISCUSSION TITLE
I would like to track the number of active connections to our cluster over time -- what's the best way to do that?

### DISCUSSION THREAD

**Discussion by spemmons** [gravitational/teleport#34347] [General]
We have servers that are remote embedded devices connected wirelessly, many of which are not on 24/7. I would like to be able to track active connections over time to determine peaks/valleys/etc.

What's the best way to do that?

Thanks!

**Comment by zmb3** [comment]
Sounds like the `teleport_connected_resources` metric is what you want.

https://goteleport.com/docs/management/diagnostics/metrics/
